projects/packages/jetpack-mu-wpcom/src/features/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-launchpad-navigator.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * Launchpad API endpoint
+ *
+ * @package automattic/jetpack-mu-wpcom
+ * @since 1.1.0
+ */
+
+/**
+ * Fetches Launchpad Navigator-related data for the site.
+ *
+ * @since 1.1.0
+ */
+class WPCOM_REST_API_V2_Endpoint_Launchpad_Navigator extends WP_REST_Controller {
+
+	/**
+	 * Class constructor
+	 */
+	public function __construct() {
+		$this->namespace = 'wpcom/v2';
+		$this->rest_base = 'launchpad/navigator';
+
+		add_action( 'rest_api_init', array( $this, 'register_routes' ) );
+	}
+
+	/**
+	 * Register our routes.
+	 */
+	public function register_routes() {
+		// Register rest route for getting a list of available checklists and the currently active checklist.
+		register_rest_route(
+			$this->namespace,
+			$this->rest_base,
+			array(
+				array(
+					'methods'             => WP_REST_Server::READABLE,
+					'callback'            => array( $this, 'get_navigator_data' ),
+					'permission_callback' => array( $this, 'can_access' ),
+				),
+			)
+		);
+	}
+
+	/**
+	 * Returns a list of available checklists and the currently active checklist.
+	 *
+	 * @return array Array with two keys: `checklists` and `active_checklist`.
+	 */
+	public function get_navigator_data() {
+		$raw_checklists = wpcom_launchpad_checklists()->get_all_task_lists();
+		$checklists     = array();
+		foreach ( $raw_checklists as $slug => $checklist ) {
+			$checklists[] = array(
+				'slug'  => $slug,
+				'title' => $checklist['title'],
```

Comment:
This value is mostly an internal value -- I think we should update this to either return a generic translated string like "Next steps for your site", or add an explicit `get_title` callbacl to the task list to get a translate value.

But we can tackle that in a follow-up.

Reply:
I think it's better for a followup